app/medInria/medMainWindow.cpp

```diff
@@ -660,6 +662,9 @@ void medMainWindow::availableSpaceOnStatusBar()
 
 void medMainWindow::closeEvent(QCloseEvent *event)
 {
+    dtkInfo() << "### Application is closing...";
+    dtkInfo() << "####################################";
```

Comment:
I added the "##########" line to easily delimit the several launches of the app. Can we just put that one (not "### Application is closing..." at the end of the function?